ivadomed/main.py

```diff
@@ -319,6 +313,16 @@ def cmd_train(context):
     epsilon = context["early_stopping_epsilon"]
     val_losses = []
 
+    metric_fns = [dice_score,  # from ivadomed/utils.py
+                  mt_metrics.hausdorff_score,
+                  mt_metrics.precision_score,
+                  mt_metrics.recall_score,
+                  mt_metrics.specificity_score,
+                  mt_metrics.intersection_over_union,
+                  mt_metrics.accuracy_score]
+    if unet_3D:
+        metric_fns.remove(mt_metrics.hausdorff_score)
```

Comment:
If we want to have hausdorff for 3D models --> hausdorff_score is implemented for 2D slices --> we could loop across the slices of the 3D patch, then return the mean. As you prefer, your call, only a suggestion.

Reply:
Yes good idea, I submitted the changes :)!